coops-ui/src/components/AnnualReport/RegisteredOfficeAddress.vue

```diff
@@ -285,38 +304,42 @@ export default class RegisteredOfficeAddress extends Vue {
   private resetAddress (): void {
     this.deliveryAddress = { ...this.deliveryAddressOriginal }
     this.mailingAddress = { ...this.mailingAddressOriginal }
+    this.emitAddresses()
     this.emitModified()
   }
 
   /**
    * Loads the office addresses from an API call.
-   *
-   * @param legalEntityNumber the identifier for the legal entity whose office addresses are fetched.
```

Comment:
Why remove this JSDoc?

Reply:
Oops. At one point I used the legalEntityNumber directly in this function (instead of passing it in), and then I reverted that code and forgot to restore the @param. I have push another commit to restore it.